src/native-common/utils/EventHelpers.ts

```diff
@@ -239,6 +233,19 @@ export class EventHelpers {
         return mouseEvent;
     }
 
+    toMouseButton(e: Types.TouchEvent): number {
+        const nativeEvent = e as any;
+        if (nativeEvent.button !== undefined) {
+            return nativeEvent.button;
+        } else if (nativeEvent.isRightButton || nativeEvent.IsRightButton) {
```

Comment:
As mentioned in previous PR, I tested current implementations and it seems that UpperCase `IsRightButton` is not produced by any native code, but I'd rather keep backwards compatibility.